src/components/InputsTable.js

```diff
@@ -119,12 +125,22 @@ const InputsTable = props => {
           setStructBuilderVisible(false)
         }
       }),
+      (!includeOptionalInputs || _.some(row => row.optional, inputTableData)) && h(div,
+        { style: { height: 0.08 * height, width, display: 'flex', alignItems: 'center' } },
```

Comment:
here's that same style definition again!

Reply:
Where would you suggest saving the style? I looked at the styles file but it doesn't look like any styles from there are used in this page. Also, this will remain as one style definition once the PRs are merged, since this style is just for the button row above the table and both buttons will be in the same row. I may abstract out the logic for showing this row into a helper function for readability once one of the PRs is merged, to make it clearer what this row is and when it's shown.